packages/eslint-plugin-template/tests/rules/no-inline-styles/cases.ts

```diff
@@ -0,0 +1,135 @@
+import { convertAnnotatedSourceToFailureCase } from '@angular-eslint/utils';
+import type { MessageIds } from '../../../src/rules/no-inline-styles';
+
+const messageId: MessageIds = 'noInlineStyles';
+
+export const valid = [
+  '<img alt="Foo eating a sandwich.">',
+  '<img src="foo" [attr.alt]="altText">',
+  `<img src="foo" [attr.alt]="'Alt Text'">`,
+  '<img src="foo" alt="">',
+  '<object aria-label="foo">',
+  '<object aria-labelledby="id1">',
+  '<object>Meaningful description</object>',
+  '<object title="An object">',
+  '<area aria-label="foo" />',
+  '<area aria-labelledby="id1" />',
+  '<area alt="This is descriptive!" />',
+  '<input type="text">',
+  '<input type="image" alt="This is descriptive!">',
+  '<input type="image" aria-label="foo">',
+  '<input type="image" aria-labelledby="id1">',
+  {
+    code: `<input [ngStyle]="{ 'background-color': '#fff' }">`,
+    options: [{ allowNgStyle: true }],
+  },
+  {
+    code: `<input [style.background-color]="'#fff'">`,
```

Comment:
Besides dash-case, I would also add an test for camelCase as mentioned [here](https://angular.io/guide/class-binding#binding-to-a-single-style).

```suggestion
    code: `<input [style.backgroundColor]="'#fff'">`,
```